src/Apps/Order/Routes/Shipping2/AddressModal2.tsx

```diff
@@ -18,11 +18,15 @@ import { AddressModalFields } from "Components/Address/AddressModalFields"
 
 import { ADDRESS_VALIDATION_SHAPE } from "Apps/Order/Routes/Shipping2/FulfillmentDetails"
 import { SavedAddressType } from "Apps/Order/Utils/shippingUtils"
-import { useShippingContext } from "Apps/Order/Routes/Shipping2/support/ShippingContext"
-import { addressWithFallbackValues } from "Apps/Order/Routes/Shipping2/support/shippingUtils"
+import { useShippingContext } from "Apps/Order/Routes/Shipping2/ShippingContext"
+import { addressWithFallbackValues } from "Apps/Order/Routes/Shipping2/shippingUtils"
 import { useCreateSavedAddressMutation$data } from "__generated__/useCreateSavedAddressMutation.graphql"
 import { useUpdateSavedAddressMutation$data } from "__generated__/useUpdateSavedAddressMutation.graphql"
 import createLogger from "Utils/logger"
+import { useCreateSavedAddress } from "Apps/Order/Routes/Shipping2/shippingContextHelpers/mutations/useCreateSavedAddress"
```

Comment:
As mentioned previously, the pattern we have in Force can be [gleaned very precisely from the example app](https://github.com/artsy/force/tree/main/src/Apps/Example) and the [best practices doc](https://github.com/artsy/force/blob/main/docs/best_practices.md). 

- mutations go in `Routes/Shipping2/Mutations/useWhateverMutation.ts` so that they are clearly discoverable and follow the rest of Force
- misc utilities go in `Shipping2/Utils/whateverUtilName.ts`

Reply:
ah sorry, missed this.

Reply:
Just noting here that I don't think the hooks or their filenames should end in mutation as that causes a naming clash with generated types 
![image](https://github.com/artsy/force/assets/9088720/b681920b-11af-497e-8f17-c71e3fca0b81)

Reply:
No prob! Its more folder structure; naming is flexible 👍